src/test/cucumber/flush.feature

```diff
@@ -37,3 +37,12 @@ Feature: Flush
     When I enter "sdk flush temp"
     Then no "res-1.2.0.zip" file is present in temporary storage
     And I see "1 archive(s) flushed"
+
+  Scenario: Clear out the cached archives and the temporary space
+    Given the archive "grails-1.3.9.zip" has been cached
+    And the file "res-1.2.0.zip" in temporary storage
+    When I enter "sdk flush all"
+    Then no archives are cached
+    And no "res-1.2.0.zip" file is present in temporary storage
+    And I see "1 archive(s) flushed"
+    And I see "1 archive(s) flushed"
```

Comment:
Should this duplicate line be here? And what does flushing temporary storage print to console?